tools/publish-release.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+if [[ `git status --porcelain` ]]; then
+  echo "Uncommited changes."
+  echo "Please commit changes and run this command again"
+  echo
+  echo "See docs/release.md for details"
+  exit
```

Comment:
```suggestion
  exit 2
```

Make this more explicitly fail?